### PR TITLE
Container and database query API's documentation refreshed with remarks

### PR DESCRIPTION
**Pull request by kirankumarkolli**
Container and database query API's documentation refreshed with remarks for single database/container read scenarios.

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/CosmosClient.cs [hunk 1]
```diff
@@ -515,6 +527,12 @@ public virtual Container GetContainer(string databaseId, string containerId)
         /// <param name="continuationToken">The continuation token in the Azure Cosmos DB service.</param>
         /// <param name="requestOptions">(Optional) The options for the item query request <see cref="QueryRequestOptions"/></param>
         /// <returns>An iterator to go through the databases.</returns>
+        /// <remarks>
+        /// Refer to http://azure.microsoft.com/documentation/articles/documentdb-sql-query/ for syntax and examples.
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/src/CosmosClient.cs]: I believe the updated URL is `https://docs.microsoft.com/azure/cosmos-db/sql-query-getting-started`

File: Microsoft.Azure.Cosmos/src/CosmosClient.cs [hunk 2]
```diff
@@ -470,6 +470,12 @@ public virtual Container GetContainer(string databaseId, string containerId)
         /// <param name="continuationToken">The continuation token in the Azure Cosmos DB service.</param>
         /// <param name="requestOptions">(Optional) The options for the item query request <see cref="QueryRequestOptions"/></param>
         /// <returns>An iterator to go through the databases.</returns>
+        /// <remarks>
+        /// Refer to https://docs.microsoft.com/en-us/azure/cosmos-db/sql-query-getting-started for syntax and examples.
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/CosmosClient.cs]: Remove en-us from the url

File: Microsoft.Azure.Cosmos/src/IDocumentClient.cs [hunk 3]
```diff
@@ -3286,7 +3286,7 @@ internal interface IDocumentClient
         /// ]]>
         /// </code>
         /// </example>
-        /// <remarks>Refer to https://msdn.microsoft.com/en-us/library/azure/dn782250.aspx and http://azure.microsoft.com/documentation/articles/documentdb-sql-query/ for syntax and examples.</remarks>
+        /// <remarks>Refer to https://msdn.microsoft.com/en-us/library/azure/dn782250.aspx and https://docs.microsoft.com/azure/cosmos-db/sql-query-getting-started for syntax and examples.</remarks>
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/IDocumentClient.cs]: Missed some links that have en-us

File: Microsoft.Azure.Cosmos/src/Linq/DocumentClientQueryable.cs [hunk 4]
```diff
@@ -845,7 +845,7 @@ public IQueryable<dynamic> CreateDocumentQuery(string collectionLink, string sql
         /// ]]>
         /// </code>
         /// </example>
-        /// <remarks>Refer to https://msdn.microsoft.com/en-us/library/azure/dn782250.aspx and http://azure.microsoft.com/documentation/articles/documentdb-sql-query/ for syntax and examples.</remarks>
+        /// <remarks>Refer to https://msdn.microsoft.com/en-us/library/azure/dn782250.aspx and https://docs.microsoft.com/azure/cosmos-db/sql-query-getting-started for syntax and examples.</remarks>
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/Linq/DocumentClientQueryable.cs]: Both URLs go to the same page. Can we remove one?
https://msdn.microsoft.com/en-us/library/azure/dn782250.aspx  
https://docs.microsoft.com/azure/cosmos-db/sql-query-getting-started

File: Microsoft.Azure.Cosmos/src/IDocumentClient.cs (hunk 3 above)
Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/IDocumentClient.cs]: DocumentClient is internal and sync will have issues.

File: Microsoft.Azure.Cosmos/src/Linq/DocumentClientQueryable.cs (hunk 4 above)
Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/Linq/DocumentClientQueryable.cs]: DocumentClientQueryable is internal.

File: Microsoft.Azure.Cosmos/src/Linq/DocumentClientQueryable.cs [hunk 5]
```diff
@@ -509,7 +509,7 @@ public IQueryable<dynamic> CreateConflictQuery(string collectionLink, SqlQuerySp
         /// </code>
         /// </example>
         /// <remarks>
-        /// The Azure Cosmos DB LINQ provider compiles LINQ to SQL statements. Refer to http://azure.microsoft.com/documentation/articles/documentdb-sql-query/#linq-to-documentdb-sql for the list of expressions supported by the Azure Cosmos DB LINQ provider. ToString() on the generated IQueryable returns the translated SQL statement. The Azure Cosmos DB provider translates JSON.NET and DataContract serialization attributes for members to their JSON property names.
+        /// The Azure Cosmos DB LINQ provider compiles LINQ to SQL statements. Refer to https://docs.microsoft.com/azure/cosmos-db/sql-query-getting-started#linq-to-documentdb-sql for the list of expressions supported by the Azure Cosmos DB LINQ provider. ToString() on the generated IQueryable returns the translated SQL statement. The Azure Cosmos DB provider translates JSON.NET and DataContract serialization attributes for members to their JSON property names.
```
Comment (khdang) [inline Microsoft.Azure.Cosmos/src/Linq/DocumentClientQueryable.cs]: >https://docs.microsoft.com/azure/cosmos-db/sql-query-getting-started#linq-to-documentdb-sql [](start = 88, length = 91)

It seems that this has moved to [https://docs.microsoft.com/en-us/azure/cosmos-db/sql-query-linq-to-sql](https://docs.microsoft.com/en-us/azure/cosmos-db/sql-query-linq-to-sql)

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/Linq/DocumentClientQueryable.cs]: DocumentClientQueryable is intenral and won't go into docs. 
This needs to be fixed in V2.

**Pull request merged**